app/uk/gov/hmrc/economiccrimelevyregistration/views/OrgAlreadyRegisteredView.scala.html

```diff
@@ -0,0 +1,31 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+        layout: templates.Layout
+)
+
+@(eclReferenceNumber: String, addLevyUrl: String)(implicit request: Request[_], messages: Messages)
+
+    @layout(pageTitle = title(messages("orgAlreadyRegistered.title"))) {
+
+        <h1 class="govuk-heading-l">@messages("orgAlreadyRegistered.heading")</h1>
+
+        <p class="govuk-body">@messages("orgAlreadyRegistered.body.p1", eclReferenceNumber)</p>
+
+        <p class="govuk-body">@Html(messages("orgAlreadyRegistered.body.p2", messages("orgAlreadyRegistered.body.p2.link", addLevyUrl)))</p>
+
+    }
```

Comment:
Similar to the controller comment - think better naming would be `GroupAlreadyEnrolledView` as well as other refs to `orgAlreadyRegistered` becoming `groupAlreadyEnrolled`